docs/Users_Guide/release-notes.rst

```diff
@@ -22,23 +27,35 @@ METplotpy Version 3.1.0-beta2 release notes (20250331)
    * Plotly line plots are plotting confidence limit bars at zero with zero length error bars (`#495 <https://github.com/dtcenter/METplotpy/issues/495>`_)
 
 .. dropdown:: Documentation
- 
+
    * Enhance METplotpy User's Guide Installation Instructions (`#457 <https://github.com/dtcenter/METplotpy/issues/457>`_)
    * Provide more background information on the Taylor Diagram (`#435 <https://github.com/dtcenter/METplotpy/issues/435>`_)
    * Enhance the Table of Contents to include all METplus components (`#499 <https://github.com/dtcenter/METplotpy/pull/499>`_)
 
 .. dropdown:: Repository, build, and test
 
-   * Update infrastructure to reflect move to developing with Python 3.12 (`#469 <https://github.com/dtcenter/METdataio/pull/469>`_)
+   * Update infrastructure to reflect move to developing with Python 3.12 (`#488 <https://github.com/dtcenter/METplotpy/pull/488>`_)
    * Update modulefiles used on various machines (`#488 <https://github.com/dtcenter/METplotpy/issues/488>`_)
-   
+
 METplotpy Version 3.1.0-beta1 release notes (20250123)
 ------------------------------------------------------
 
+.. dropdown:: New Plots
```

Comment:
Suggest removing any with "None" to better match the other METplus components. In Slack on 3/25, I sent the following note: 

> As you know, I always put in a title "Bugfixes", "Enhancements", etc, even if there are none and I put "None" underneath.  I see in the MET and METplus repos they just don't put the title "Bugfixes", "Enhancements", etc, if that category had no changes.  I guess I'll start doing the same for consistency...

Reply:
This is inconsistent with how thing were originally done.  Initially, when there was nothing, the pull down was left empty.  Then, the practice was to add 'None' to the pull down.

Reply:
Yes, I remember.  It looks like now METplus and MET are simply not including the title if changes for that title are not included, so I followed suit for the METcalcpy and METdataio releases and thought METplotpy and METviewer should do the same for consistency.  Please feel free to add something to the meeting agenda for tomorrow if you'd like to discuss it.  I don't have any strong preferences, except a preference for consistency.